benchmarks/bm_runner.py

```diff
@@ -313,7 +313,7 @@ class _SubParserGenerator(ABC):
     description: str = NotImplemented
     epilog: str = NotImplemented
 
-    def __init__(self, subparsers: ArgumentParser.add_subparsers) -> None:
+    def __init__(self, subparsers) -> None:
```

Comment:
`argparse._SubParsersAction[ArgumentParser]`, but that is a private type so I thought it might be best not to use it, you never know when `argparse` decides to change its internals.

Reply:
Yes, a little risky, but I would like this included as the code I have written is a little strange! Thank you for finding the correct type hint.

Reply:
Done in 74d32b9dd7c340239b3a390d9d7c616d02e02287